.github/actions/gradle-setup/action.yml

```diff
@@ -18,6 +18,15 @@ runs:
         path: RegistrationService
         ref: ef045e3b6556bf1b1148536a1be30d564fd56bf6
 
+    - name: Checkout Identity Hub
+      uses: actions/checkout@v2
+      with:
+        # TODO: switch to upstream repository once changes are merged
+        #repository: eclipse-dataspaceconnector/IdentityHub
+        repository: agera-edc/IdentityHub
```

Comment:
I would move the sources out, to ensure no side effect (also in the future). See as done for RegistrationService

Reply:
I decided to delete the sources as I don't think we need them